#Goal Selection Param File ~#

#~ General Parameters
slam_map: /map                              # SLAM map topic
coverage_map: /coverage_map                 # Coverage map topic
map_frame: map                              # Map frame
base_footprint_frame: base_footprint        # Base footprint frame
# velocity pub topic
velocity_pub: /mobile_base/commands/velocity
# resolution used to be 0.05
# change also delta param according in
# gmapping.launch.xml in turtlebot
# navigation package
resolution: 0.1                 # Map resolution

#num_of_sensors: 1
#
#dist_of_use_1: 3
#hfov_1: 360
sensor_frame_1: base_footprint


